src/services/test-result.ts

```diff
@@ -0,0 +1,77 @@
+import { Callback } from 'aws-lambda';
+import createResponse from '../utils/create-response';
+import { DynamoDB, AWSError } from 'aws-sdk';
+import * as UUID from 'uuid'
+import { ITestResult, IResponse } from '../interfaces/interface';
+
+export default class TestResultService {
+
+	constructor(private db: DynamoDB.DocumentClient, private tableName: string) { }
+	
+	create(requestBody: any, callback: Callback) {
```

Comment:
Probably this should be addToDatabase? create can mean lots of things...
If TestResultService is a crud service then create is fine, we should rename it TestResultCrudService though

Reply:
hym, TestResultService.create is ok I think. Wdut?

Reply:
Yes, it's just not clear it adds to the database (it could be creating an object in memory) - leave it for now, we can look at it later